src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java

```diff
@@ -1,12 +1,12 @@
 /**
- * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
  *
  * This file is part of fiware-cygnus (FI-WARE project).
  *
- * cosmos-injector is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
```

Comment:
I noticed that there are many other references to "cosmos-injector" in this branch. Although I guess it is not the most important part of this PR, maybe an issue to fix all these references is needed. I really do not know if it is already created, considered or directly should not be fixed :-) If you would like me to create this issue, please just tell ;) Thanks!